conjure-java-core/src/main/java/com/palantir/conjure/java/services/JerseyServiceGenerator.java

```diff
@@ -288,7 +291,8 @@ private MethodSpec createCompatibilityBackfillMethod(
         Optional<AuthType> auth = endpointDef.getAuth();
         createAuthParameter(auth, withAnnotations).ifPresent(parameterSpecs::add);
 
-        ParameterOrder.sorted(endpointDef.getArgs()).forEach(def ->
+        boolean disableParameterSorting = featureFlags.contains(FeatureFlags.DisableParameterSorting);
+        ParameterOrder.sorted(endpointDef.getArgs(), disableParameterSorting).forEach(def ->
```

Comment:
We should rename this utility function to `ordered`, it's odd to say `sort(<values>, dontReallySort)`